CONTRIBUTING.md

```diff
@@ -9,47 +9,50 @@ is welcome to do so.
 This document aims at establishing guidelines for contributions within
 Lethe.
 
-Lethe has a wiki which is openly accessible on : https://github.com/chaos-polymtl/lethe/wiki
+Lethe has a wiki which is openly accessible on : https://chaos-polymtl.github.io/lethe/index.html
 
 
 # How can I contribute?
 
 Contributions through code or documentation should be done through pull
 request at the official Github repository of Lethe : https://github.com/chaos-polymtl/lethe.
 We recommend that users either request access to the Lethe repository or
-create their own fork of Lethe on their own github account. This can then be
+create their own fork of Lethe on their own Github account. This can then be
 used to open pull requests
 
 # What should be the content of a pull request (PR)?
 
 A pull request should contain the following elements:
 
 - A brief title (less than 60 characters) that describes the goal of the pull
-requests.
+request.
 - A detailed description of the content added by the pull request:
-i) If the PR adds a new feature: The feature should be documented, tests (with
-  a unit tests and/or an application test) and if the feature adds or alters
-  parameters in the input file, the appropriate section of the wiki should
-  be updated accordingly.
-ii) If the pull requests corrects a bug: the source of the bug should be
-explained and the way it was identified should be briefy described. A unit
-test or application test that reproduces the bug should be added.
+  * If the PR adds a new feature:\
+    The feature should be documented, tested (with unit tests and/or application tests) 
+    and if the feature adds or alters parameters in the input file, the appropriate section 
+    of the documentation should be updated accordingly.
+  * If the pull request corrects a bug:\
+    the source of the bug should be explained and the way it was identified should be briefly described. 
+    A unit test or application test that reproduces the bug should be added.
+
+See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.
+
 
 # Good practices
 
 - Before making a pull request, the branch should be rebased on master to ensure
-a linear history and to make merging as easy as possible. It is the responsability
+a linear history and to make merging as easy as possible. It is the responsibility
 of the branch owner to ensure that the pull request goes as smoothly as possible.
-Pushing to github after a rebase requires a force push. You can accomplish
+Pushing to Github after a rebase requires a force push. You can accomplish
 this by using `git push --force-with-lease`
 - All functions should be documented in the `.h`. All functions should contain
 a `@brief` that describes the function and a `@param` for each argument.
```

Comment:
Unrelated:
```suggestion
- All functions should be documented in the `.h`. All functions should contain
a `@brief` that describes the function, a `@param` for each argument, and a `@return` if anything is returned by the function.
```

Reply:
I did not know we needed the @return actually. I guess our "official" Doxygen rules should be all listed in the contributing doc and not here